meetings/2022-05-02.md

```diff
@@ -0,0 +1,118 @@
+* AvdB: if it is required that an OP is required to dereference these client_id URIs, then it makes interop much easier to achieve.
+* eP: the only case when it's not a client identifier, it's an identifier issued by the OP, which will be pre-registered by the OP. The OP can then just check if it's one of these other formats, it can use that; otherwise, it can just dereference the URI as a client identifier.
+* AC: This is how I implemented it.
+* eP: even for static registration, an OP will still manage these identifiers
+* AC: This is exactly my experience with implementation. The identifier is either one that was created by the OP, or an external one which would be a Client ID IRI.
+
+### UMA updates
+
+* LD: I am working with the CSS project, having trouble with DPoP during the claims pushing stage. This is unclear on the Solid-OIDC side. This could be an opportunity to specify an IRI for a DPoP-bound ID token.
+* AC: The way I've done this, in one stage you may be providing ID Token. When you push that claim it may perform validation of that ID Token. Also part of this validation is validating the DPOP binding. The part of this interaction involves DPoP header. The format is definitelly OIDC ID Token with some extra claims. One could argue that it is ID Token but also an argument could be made that it is a different claim format.
+* LD: My problem is mainly getting references to UMA and DPoP as normative references. DPoP defines use with access token.
+* LD: UMA doesn't define the use of DPoP, then there is nothing in the request body to tell the AS how to handle these JWTs
+* LD: In UMA, we're using an IRI but there is no link to DPoP
+* eP: We have discussed having Solid-OIDC UMA as a separate specification document where this could be normatively defined along with all of the UMA details.
+* LD: That would be a very good idea, which could enable other types of claims that could be provided during the claims pushing stage
+* AC: One thing that we are trying to do with UMA is normatively require use of the Authorization Server, but we don't want to limit it to UMA. There may be different approaches to Authorization Server. We try to balance those two competing needs. If one uses UMA we want to provide all the requrirements.
+* LD: I think Pavlik's proposal makes sense. UMA details can be in dedicated document. It could also specify the details about DPoP-bound ID Token with webid claim.
+* AC: this makes a lot of sense and it really valuable feedback
+
+### Alternative flows
+
+* LD: One small question about the status of the alternative flow document
+* AC: We have been busy with publishing FPWD of Solid-OIDC. I think defining alternative flows is next priority. Defining it on OIDC level it may be problematic. Client credentials are OAuth2 flow not OIDC.
```

Comment:
```suggestion
* AC: We have been busy with publishing FPWD of Solid-OIDC. I think defining alternative flows is next priority. Defining it on OIDC level it may be problematic. Client credentials are OAuth2 flow, not OIDC.
```